From 17d27a39b8542cb16cb18335ea490944bd63c54b Mon Sep 17 00:00:00 2001
From: Eli Schwartz <eschwartz93@gmail.com>
Date: Wed, 16 May 2018 15:24:11 -0400
Subject: [PATCH] Remove unnecessary console script

The purpose of this  module is to cache things, not to test if they
*can* be cached. This clutters up the $PATH.
---
 setup.py | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/setup.py b/setup.py
index 07f8b5c..79ca493 100644
--- a/setup.py
+++ b/setup.py
@@ -25,11 +25,6 @@ setup_params = dict(
         'filecache': ['lockfile>=0.9'],
         'redis': ['redis>=2.10.5'],
     },
-    entry_points={
-        'console_scripts': [
-            'doesitcache = cachecontrol._cmd:main',
-        ]
-    },
     classifiers=[
         'Development Status :: 4 - Beta',
         'Environment :: Web Environment',
-- 
2.17.0

